### DISCUSSION TITLE
Is it possible to convert a GraphQL Schema to SQL create table queries?

### DISCUSSION THREAD

**Discussion by RageshAntony** [hasura/graphql-engine#9317] [Question]
My Free Heroku DBs that connected with Hausra was deleted, I am trying to recover it.

Before that, I am curious whether it is possible to generate SQL Create table queries from an existing GraphQL schema file ... That is, is it possible to create the database with tables with relations (datas not needed) from the GraphQL schema file ?

**Comment by adas98012** [comment]
@RageshAntony 
It is not possible to generate "Create Table" queries from an existing GraphQL schema file in Hasura. Hope, you'll be able to recover your DBs in an alternate way.

Thanks
